src/index.js

```diff
@@ -82,3 +73,15 @@ export default class DesignSystem {
     return this.get(color, this.ds.colors.brand)
   }
 }
+
+// Converts px to rem/em
+export const pxTo = (value, base = 16, unit = 'rem') =>
+  `${parseFloat(value) / base}${unit}`
+
+// Converts rem/em to px
+export const toPx = (value, base = 16) => `${parseFloat(value) * base}px`
+
+export const parseUnit = str => {
+  str = String(str)
```

Comment:
Isn't the `str` parameter already a String? If so, then it's unnecessary to cast it to String again. If not, then perhaps you should change the name of the `str` parameter, as it would be misleading.

Reply:
you are right, I will update to use a better implementation